monkestation/code/modules/cybernetics/implant_items/weapons/mantis_blade.dm

```diff
@@ -0,0 +1,87 @@
+/obj/item/mantis_blade
+	name = "C.H.R.O.M.A.T.A. mantis blade"
+	desc = "Powerful inbuilt blade, hidden just beneath the skin. Singular brain signals directly link to this bad boy, allowing it to spring into action in just seconds."
+	icon_state = "mantis"
+	inhand_icon_state = "mantis"
+	icon = 'monkestation/code/modules/cybernetics/icons/items_and_weapons.dmi'
+	lefthand_file = 'monkestation/code/modules/cybernetics/icons/swords_lefthand.dmi'
+	righthand_file = 'monkestation/code/modules/cybernetics/icons/swords_righthand.dmi'
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	flags_1 = CONDUCT_1
+	force = 12
```

Comment:
An issue with the mantis blades, 12 damage is worse than the circular saw or welder that surgical/toolset implants give you.
When using two mantis blades you use both at once so it'll be stronger, but if you decide you want one arm to be a surgical arm or something, then if you decide use a single mantis blade, it won't be as strong as a dedicated combat implant should.

Reply:
Well yeah. You're meant to use two or none really.